$OpenBSD: patch-renderer_qgl_h,v 1.1.1.1 2019/09/02 18:04:40 thfr Exp $

include OpenBSD in definitions
absolute path needed for gl.h?? and glx.h??
- but that leads to problems down the line

Index: renderer/qgl.h
--- renderer/qgl.h.orig
+++ renderer/qgl.h
@@ -29,7 +29,7 @@
 #define GL_GLEXT_LEGACY
 #include <OpenGL/gl.h>
 
-#elif defined( __linux__ )
+#elif defined( __linux__ ) || defined(__OpenBSD__)
 
 // using our local glext.h
 // http://oss.sgi.com/projects/ogl-sample/ABI/
